generate_stats.py

```diff
@@ -53,8 +71,12 @@ def collect_statistics(my_core, root_set):
     stats = {}
     dagmc_tags = dagmc_stats.get_dagmc_tags(my_core)
     entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
-    stats['Native Ranges'] = dagmc_stats.get_native_ranges(my_core, root_set, entity_types)     # get Ranges of various entities
-    stats['EntitySet Ranges'] = dagmc_stats.get_entityset_ranges(my_core, root_set, dagmc_tags['geom_dim']) 
+    native_ranges = dagmc_stats.get_native_ranges(my_core, root_set, entity_types)     # get Ranges of various entities
```

Comment:
This is mostly a matter of preference, but for readability, I'd divide this block up with carefully placed blank lines.  Think of it like dividing prose up into paragraphs.  Some lines belong together because they are related but in blocks they can be separated somewhat.  For example, you could have a "block" for each of the following:
* getting the tags
* getting the native ranges
* getting the geometry set ranges
* getting and processing the s_p_v data